src/distance_finder.cpp

```diff
@@ -75,6 +84,13 @@ class DistanceFinder {
     double angle_error_adjustment_;                 //!< used to expand the angles provided from bounding boxes
     double max_lidar_range_;
     double lidar_position_;
+    int min_circle_pts_;
+    double marker_radius_;
```

Comment:
This is new, but going forward we are naming parameters with postfix "_p" so `marker_radius_` becomes `marker_radius_p`